### DISCUSSION TITLE
Π2019223/ Παραδοτέο 3

### DISCUSSION THREAD

**Discussion by PetroulaStylianou** [courses-ionio/sw#1329] [Show and tell]
Στο τρίτο παραδοτέο μας ζητήθηκε η εγκατάσταση ενός λογισμικού χωρίς systemd. Για το σκοπό αυτό επέλεξα την χρήση του void linux με το οποίο πραγματοποίησα την άσκηση γραμμής εντολών που εκτέλεσα και στο προηγούμενο παραδοτέο.

[Αναφορά](https://github.com/PetroulaStylianou/sw/blob/2019223/projects/2019223/README.md)

**Comment by voltmaister** [comment]
Καλησπέρα συνάδελφε, καλό θα ήταν να βάλεις και ένα asciinema με ένα neofetch στο παραπάνω discussion.
